browser/ui/tabs/brave_tab_menu_model.cc

```diff
@@ -166,3 +177,29 @@ void BraveTabMenuModel::BuildItemsForSplitView(
                              IDS_IDC_SWAP_SPLIT_VIEW);
   }
 }
+
+#if BUILDFLAG(ENABLE_CONTAINERS)
+void BraveTabMenuModel::OnContainerSelected(int container_id) {
+  // Should open selected tabs in the specified container.
+  // TODO(sko) Questions:
+  // Q1. What should we do if there's multiple tabs selected?
+  // Q2. What should we do if user selects already selected container? Are we
+  //     going to take the tabs out of the container, or do nothing?
+  NOTIMPLEMENTED();
+
+  // In-development code to simulate the container selection.
+  MockContainersService::GetInstance().set_selected_container_id(container_id);
```

Comment:
The container is a state of a tab, we will have most likely some tab helper that will return a current container.